sdk/log/simple_test.go

```diff
@@ -69,30 +68,6 @@ func TestSimpleProcessorForceFlush(t *testing.T) {
 	require.True(t, e.forceFlushCalled, "exporter ForceFlush not called")
 }
 
-func TestSimpleProcessorConcurrentSafe(t *testing.T) {
```

Comment:
Why is this removed? The processor needs to ensure its methods are concurrent safe.

Reply:
Why if exporter is required to be concurrent safe?

Reply:
Should I test it with a concurrent -safe exporter?

Reply:
I'm not sure how the exporter being required to be concurrent safe removes the requirements that the simple log processor needs to also be concurrent safe. Can you explain?

Reply:
OK I will update to have a test which uses a concurrent-safe exporter.

Reply:
[This](https://github.com/open-telemetry/opentelemetry-go/blob/bc9c928abd1129d8a8f301b53c15472fa8624d2f/sdk/log/exporter.go#L57-L63) isn't concurrent safe?!

Reply:
5bc5cae8 😄